src/state/migrations/index.ts

```diff
@@ -122,4 +122,6 @@ export const migrations = {
   111: clearAssets,
   112: clearAssets,
   113: clearAssets,
+  114: clearAssets,
+  115: clearAssets,
```

Comment:
```suggestion
```

Reply:
Fixed in [2c31e70](https://github.com/shapeshift/web/pull/7798/commits/2c31e70f633ec09d24a948403a2e9819a760ff36)